third_party/bazel/acl/acl_makefile.patch

```diff
@@ -14,4 +14,11 @@ index 0000000..bd9a183
 +	mkdir -p $(INSTALLDIR)/build $(INSTALLDIR)
 +	cp build/libarm_compute.a build/libarm_compute_core.a build/libarm_compute_graph.a $(INSTALLDIR)/build/
 +	cp -r build/src/core/arm_compute_version.embed arm_compute/ src/ utils/ support/ include/ $(INSTALLDIR)
+diff --git a/build/.gitignore b/build/.gitignore
+new file mode 100644
+index 000000000..48cdce852
+--- /dev/null
++++ b/build/.gitignore
+@@ -0,0 +1 @@
++placeholder
```

Comment:
what's this for?

Reply:
Just make `build` dir exists before build happens, otherwise bazel build will failed.